atc/steps.go

```diff
@@ -158,13 +167,13 @@ type StepConfig interface {
 }
 
 // StepWrapper is an optional interface for step types that is implemented by
-// steps that wrap other steps (e.g. hooks like `on_success`, `timeout`, etc.)
+// steps that wrap/modify other steps (e.g. hooks like `on_success`, `timeout`, etc.)
 type StepWrapper interface {
 	// Wrap is called during (Step).UnmarshalJSON whenever an 'inner' step is
 	// parsed.
 	//
-	// Modifier step types should implement this by assigning to an internal
-	// field that has a `json:"-"` struct tag.
+	// Modifier step types should implement this function by assigning to an
+	// internal field, that has a `json:"-"` tag, the passed in StepConfig.
```

Comment:
I think putting 'the passed in StepConfig' all the way end makes this flow a little awkwardly, haha. Sorry for the nitpick. I'm no authority on English so I'll still approve anyway.

Reply:
I agree, I think it reads better if we reference the StepConfig earlier. Changed it!